.github/workflows/e2e.yaml

```diff
@@ -8,13 +8,21 @@ on:
     branches:
       - main 
     paths:
+      # include test files in e2e again
+      - "test/**" 
       - "Dockerfile"
       - "**.go"
-      - "!**_test.go" # exclude test files to ignore unit test changes
-      - "test/**" # include test files in e2e again
       - ".github/workflows/e2e.yaml"
       - "charts/**"
       - "manifests/**"
+      # exclude test files to ignore unit test changes
+      - "!**_test.go"
+      - "!**.md"
+      - "!devspace.yaml"
+      - "!devspace_start.sh"
+      - "!LICENSE"
+      - "!.vscode"
+      - "!**.toml"
```

Comment:
I don't think this is correct. If I understand it correctly, changes to `test/e2e/e2e_suite_test.go` would be ignored, which is not desired.
Also, some entries are not necessary because we do not include project root in the `paths` , so these are not needed IMHO - `"!devspace.yaml"`, `"!devspace_start.sh"`, `"!LICENSE"`,  `"!.vscode"`, `"!**.toml"`

Reply:
Ahh, I forgot the order of the paths, to include `test/e2e/e2e_suite_test.go`, `test/**`  should be defined after `!**_test.go`.